assets/css/main.css

```diff
@@ -235,6 +235,11 @@ body {
   color: var(--main-color);
   background: none;
 }
+
+.main-head h1  {
+  font-size: 3rem;
```

Comment:
There's already a line that sets the font size in this file for `.main-head h1` so please change that instead of adding a new one

Reply:
sorry ...

Reply:
I see you removed your change and kept the old value. Can you please modify the old value?